diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter1/IdentifierNaming.kt

```diff
@@ -139,6 +145,36 @@ class IdentifierNaming(configRules: List<RulesConfig>) : DiktatRule(
         return false
     }
 
+    /**
+     * method checks that identifier is correct backing field
+     */
+    private fun ASTNode.isCorrectBackingField(variableName: ASTNode): Boolean {
+        val propertyNodes = this.treeParent.getAllChildrenWithType(KtNodeTypes.PROPERTY)
+        val variableNameCut = variableName.text.drop(1)
+        // check that backing field name is correct
```

Comment:
I have seen this code somewhere already... Why backing fields logic was touched here?

Reply:
Deleted